ansible/roles/commcarehq/templates/logrotate.conf.j2

```diff
@@ -0,0 +1,29 @@
+# see "man logrotate" for details
+# rotate log files weekly
+weekly
+
+# packages drop log rotation information into this directory
+include /etc/logrotate.d
+
+# no packages own wtmp, or btmp -- we'll rotate them here
+/var/log/wtmp {
+    missingok
+    monthly
+    create 0664 root utmp
+    rotate 1
+}
+
+/var/log/btmp {
+    missingok
+    monthly
+    create 0660 root utmp
+    rotate 1
+}
+
+{{ log_home }}/*.log {
+    weekly
+    missingok
+    compress
+    delaycompress # compress on second rotation
+    olddir {{ rotated_log_home }}
```

Comment:
In other places, we just leave these in the same directory (though maybe you're saying that's not the best plan). One nice thing about that is you're in there grep'ing, realize that what you're looking for is more than a month ago, and then you can just sandwich your previous command with `z`...`.1` and run it again (zgrep for grep'ing gzipped files). That's such a weak argument in absolute terms that if you have any sort of reason to do this it probably trumps that.